index.html

```diff
@@ -337,15 +337,15 @@ <h3>Language</h3>
         <ul class="html">
           <li>
             <img src="Icon and img/Html.png" alt="" />
-            <p>Html</p>
+            <p class="lettering">Html</p>
           </li>
           <li>
             <img src="Icon and img/JavaScrispt.png" alt="" />
-            <p>Javascript</p>
+            <p class="lettering">Javascript</p>
           </li>
           <li>
             <img src="Icon and img/Css.png" alt="" />
-            <p>CSS</p>
+            <p class="lettering">CSS</p>
           </li>
```

Comment:
- [ ] The width for the 2 tags as attached in the image should be 40% and 60% respectively for the about me and the languages, kindly adapt it to your own too so it would follow the design to style.

### Your design
![image](https://github.com/tchasinga/Setup-first-mobile/assets/86200356/2e0536e9-ca94-4f69-a1c8-0c726cc67127)


### Figma
![image](https://github.com/tchasinga/Setup-first-mobile/assets/86200356/0cc0907a-32cc-4f0f-a7fd-55f435bcabca)

Reply:
- [ ] As mention by the previous reviewer, There are 2 blocks  in the "About me" section, kindly give 40% of the pages width to left block and  60% of the page's width to the right block.

**HIT: ** Use the `grid-template-columns: 40%, 60%`

> * [ ]  The width for the 2 tags as attached in the image should be 40% and 60% respectively for the about me and the languages, kindly adapt it to your own too so it would follow the design to style.
> 
> ### Your design
> ![image](https://user-images.githubusercontent.com/86200356/237632660-2e0536e9-ca94-4f69-a1c8-0c726cc67127.png)
> 
> ### Figma
> ![image](https://user-images.githubusercontent.com/86200356/237632726-0cc0907a-32cc-4f0f-a7fd-55f435bcabca.png)

- [ ] At `910px` of screen width the "About Myself" section is distort. Kindly make sure it responsive on all screen size.
![Screenshot from 2023-05-11 11-20-43](https://github.com/tchasinga/Setup-first-mobile/assets/91142432/6b5fcf8a-4425-48cb-a0d7-0614d5259e62)


- [ OPTIONAL] Note the `font-size` of Languages, Frameworks, and Skills information in the block at the right in your design it different from the Figma design. Kindly make sure you stick to the Figma design as much as possible.
### Your design :point_down: 
![Screenshot from 2023-05-11 12-03-08](https://github.com/tchasinga/Setup-first-mobile/assets/91142432/7d12ee11-b23d-436d-953c-ef902da73af3)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-11 12-05-56](https://github.com/tchasinga/Setup-first-mobile/assets/91142432/fc5156a3-08d9-4df8-a850-30cefaeec1c5)
_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_